components/TextAreaWithCheckbox.tsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import {
+  DynamicComponentControlledProps,
+  DynamicComponent
+} from "remultiform/component-wrapper";
+import {
+  Checkboxes as LBHCheckboxes,
+  CheckboxItem
+} from "lbh-frontend-react/components";
+import { Textarea } from "lbh-frontend-react/components/Textarea";
+import PropTypes from "../helpers/PropTypes";
+
+export type TextAreaWithCheckboxProps = DynamicComponentControlledProps<{
+  value: string;
+  isPostVisitAction: boolean;
+}> & {
+  label: {
+    id?: string;
+    value: React.ReactNode;
+  };
+  name: string;
+  rows?: number;
+  includeCheckbox?: boolean;
```

Comment:
Would this ever be `false`? I think we don't want this prop, and instead always add the checkbox (the same says "with checkbox", so having it be optional is a bit weird.